Example/Example.xcodeproj/project.pbxproj

```diff
@@ -367,20 +367,21 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-
-				DEVELOPMENT_TEAM = C65W8JU69P;
-
+				CODE_SIGN_IDENTITY = "Apple Development";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
+				CODE_SIGN_STYLE = Manual;
+				DEVELOPMENT_TEAM = "";
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = 4R67Q8JL33;
 				INFOPLIST_FILE = "$(SRCROOT)/Sources/Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-
-				PRODUCT_BUNDLE_IDENTIFIER = com.zr0z.Example;
-
+				PRODUCT_BUNDLE_IDENTIFIER = Bacancy.virtusizeDemoApp.com;
```

Comment:
What is the reason to have Demo in the `virtusizeDemoApp` here ?
From my understing it is means to be release in production so it is not a demo right ?

Reply:
While we are developing the SDK, the demo is where we test the SDK in the demo app. For the current update, we also need to test the SDK in the demo app since we need to archive the demo to verify that the manifest file is correct. any mistakes made during archiving

Reply:
OK.